Python/BMICalculator/bmi-calculator.py

```diff
@@ -0,0 +1,27 @@
+from colorama import Fore, init
+
+init(autoreset=True)
+
+feet, inches = map(int, input(Fore.CYAN + "\nEnter height in feet and inches (example 5'3''): ")[:-2].split("'"))
```

Comment:
@Ninjavin Instead of feet and inches Use: Enter your height? (inches or meters)  : 5.3
It will become more simple.